sorl/thumbnail/engines/pil_engine.py

```diff
@@ -277,7 +277,7 @@ def _get_raw_data(self, image, format_, quality, image_info=None, progressive=Fa
             # Do not save unnecessary exif data for smaller thumbnail size
             params.pop('exif', {})
             image.save(bf, **params)
-        except (IOError, OSError):
```

Comment:
Python 3 aliased IOError to OSError so they are essentially the same exception

flake8 threw a warning on this